FosServer/src/main/java/com/example/fostracker/models/Address.java

```diff
@@ -0,0 +1,66 @@
+package com.example.fostracker.models;
+
+public class Address {
```

Comment:
Please add a javadoc comment.
https://google.github.io/styleguide/javaguide.html#s7-javadoc

Reply:
do you want it for just the address class or all classes?

Reply:
added.